.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,12 +26,20 @@
 
 ## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
 
+## Addeds tests?
```

Comment:
`Addeds` should be `Added`

Reply:
Thanks for catching this @jacobherrington!